### PR TITLE
Specify "magic" variables' type only in component tooltip, not via prefix

### PR DESCRIPTION
**Pull request by gnidan**
## PR description

This updates the `MagicValue` component to omit the prefix specifying which kind of magic variable it is, instead putting that information in the tooltip.

See: 

Screenshot 2023-06-21 at 11 06 17 PM

### CODE REVIEW

**Comment by gnidan** [comment]
cc @eggplantzzz so you're aware

**Comment by haltman-at** [comment]
Seems sensible to me!  Although I'm a bit wary to approve it without @cliffoo. :P

File: packages/codec-components/src/react/components/codec/format.values.magic-value.tsx [hunk 1]
```diff
@@ -15,9 +15,9 @@ export const { MagicValue } = createCodecComponent(
       <Container
         prefix={
           <>
-            <Code title="type: magic (debugger-only)">{type.variable}</Code>
-            <Code type="colon">:</Code>
-            <Code type="bracket">{"{"}</Code>
+            <Code title={`type: ${type.variable}`} type="bracket">
+              {"{"}
+            </Code>
           </>
```
Comment (cliffoo) [inline packages/codec-components/src/react/components/codec/format.values.magic-value.tsx:21]: No need for the `<></>` fragment anymore!

**Review by cliffoo — APPROVED** [review]
👍

**Review by cliffoo — CHANGES_REQUESTED** [review]
Oops almost forgot, need injected nodes! See [here](https://github.com/trufflesuite/truffle/commit/84225a69ad86a56c7b89357a36f793e137c56898).

**Review by cliffoo — APPROVED** [review]

**Pull request merged**